src/coreclr/vm/domainfile.h

```diff
@@ -587,6 +587,8 @@ class DomainAssembly : public DomainFile
     void Allocate();
     void DeliverSyncEvents();
     void DeliverAsyncEvents();
+    void RegisterWithHostAssembly();
+    void UnRegisterFromHostAssembly();
```

Comment:
`UnRegister` -> `Unregister`

Reply:
Thanks! I will do the rename in some other change.